### ISSUE TITLE
5.10.0 unable to start with driver error

### ISSUE THREAD

**Issue by jurgenweber** [AlCalzone/node-zwave-js#3595]
### Version

[zwave_js_dump.json.zip](https://github.com/zwave-js/zwavejs2mqtt/files/7416514/zwave_js_dump.json.zip)

**Checklist:**

- [ ] I am **not** using Home Assistant. **Or:** a developer has told me to come here.
- [x] I have checked the troubleshooting section and my problem is **not** described there.
- [x] I have read the changelog and my problem is **not** mentioned there.

While I am using home assistant, thnis has nothing to do with it. The server is failing before HA gets involved.

Build/Run method

- [x] Docker
- [ ] PKG
- [ ] Snap package
- [ ] Manually built (git clone - yarn install - yarn run build )

Zwavejs2Mqtt version: 5.10.0
Z-Wave JS version: I can not get to the control panel, it doesn't start.

### Describe the bug

the server does not start.

```
| 2021-10-26 18:33:22.972 INFO APP: Version: 5.10.0
| 2021-10-26 18:33:22.979 INFO APP: Application path:/usr/src/app
|
......
|
| 2021-10-26 18:33:23.042 WARN STORE: scenes.json not found
| 2021-10-26 18:33:23.079 INFO APP: Listening on port 8091 host 0.0.0.0 protocol HTTP
| 2021-10-26 18:33:23.138 INFO MQTT: MQTT is disabled
| 2021-10-26 18:33:23.159 INFO ZWAVE: Connecting to /dev/ttyACM0
| 2021-10-26 18:33:23.223 INFO ZWAVE: Zwavejs usage statistics ENABLED
| Warning: Event "unpause" was sent to stopped service "SendThread". This service has already reached its final state, and will not transition.
| Event: {"type":"unpause"}
| 2021-10-26 18:33:54.108 INFO ZWAVE: Controller status: Driver: Failed to initialize the driver: ZWaveError: The driver is not ready or has been destroyed (ZW0103)
|     at Driver.ensureReady (/usr/src/app/node_modules/zwave-js/src/lib/driver/Driver.ts:1764:10)
|     at Driver.sendMessage (/usr/src/app/node_modules/zwave-js/src/lib/driver/Driver.ts:2931:8)
|     at ZWaveController.interview (/usr/src/app/node_modules/zwave-js/src/lib/controller/Controller.ts:506:22)
|     at Driver.initializeControllerAndNodes (/usr/src/app/node_modules/zwave-js/src/lib/driver/Driver.ts:932:27)
|     at Immediate.<anonymous> (/usr/src/app/node_modules/zwave-js/src/lib/driver/Driver.ts:809:16) (ZW0100)
| 2021-10-26 18:33:54.111 INFO GATEWAY: Driver is CLOSED
| 2021-10-26 18:33:54.113 INFO ZWAVE-SERVER: Server closed
| 2021-10-26 18:33:54.114 INFO ZWAVE: Client closed
| 2021-10-26 18:33:54.115 INFO GATEWAY: Driver is CLOSED
| 2021-10-26 18:33:54.123 INFO ZWAVE: Connecting to /dev/ttyACM0
| 2021-10-26 18:33:54.137 INFO ZWAVE: Zwavejs usage statistics ENABLED
```

### To Reproduce

upgrade to 5.10.0, restart server.

### Expected behavior

I expect it to start as per 5.9.0.

### Additional context

I thought at first I had a hardware issue, I restarted the server, usb sticks, cables, replugged, etc. Same result, revert to 5.9.0 and it works fine again.

**Comment by gensyn** [comment]
I have the same problem with Docker and a ConBee II device. I also reverted back to 5.9.0 to get it working again.

**Comment by robertsLando** [comment]
> | 2021-10-26 18:33:23.159 INFO ZWAVE: Connecting to /dev/ttyACM0

Could you try please to use `/dev/serial/by-id` path instead of `/dev/ttyACM0`?

**Comment by gensyn** [comment]
I changed my docker device bindings from `/dev/serial/by-id/XXX:/dev/ttyACM0` to `/dev/serial/by-id/XXX:/dev/serial/by-id/XXX` and changed the port in the settings (which worked fine). But after updating to 5.10.0 I'm still getting the same error.

**Comment by AlCalzone** [comment]
> ConBee II

That's Zigbee, not Z-Wave?!

Which Z-Wave sticks do you guys have? UZB1 by any chance? If yes, then the `serial/by-id` paths don't seem to work either and you need to disable soft reset in the zwavejs2mqtt options.

**Comment by gensyn** [comment]
Shoot, you are right. For Z-Wave I'm using a Z-Wave.Me UZB Smart Home Stick. At a quick glance I couldn't find any options regarding soft reset though.

**Comment by AlCalzone** [comment]
> Z-Wave.Me UZB Smart Home Stick

Then I'm guessing you also have the UZB**1**

The option is there:
![grafik](https://user-images.githubusercontent.com/17641229/138879289-e8d28de3-1b28-4fec-9be8-50726d00fd41.png)

**Comment by gensyn** [comment]
That screen looks different for me:

![grafik](https://user-images.githubusercontent.com/36128035/138880184-cc389015-ff80-4bf9-8f36-5c81d28d6954.png)

**Comment by AlCalzone** [comment]
Are you on z2m 5.10 again?

**Comment by gensyn** [comment]
No, I'm still on 5.9.0 as 5.10.0 was throwing the error. I actually didn't check if the settings were available in 5.10.0 since I figured the error would crash everything but I'm trying now.

**Comment by gensyn** [comment]
Ok, diasbling Soft Reset fixed the issue for me. Thanks for the support.
